src/components/SearchBar.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+
+const SearchBar = () => {
+
+    const [query, setQuery] = useState('');
+
+    const handleInputChange = (event) => {
+        const value = event.target.value;
+        setQuery(value);
+    };
+
+    return (
+        <div className="search-bar">
+            <input
+                type="text"
+                placeholder="What's on your plate today? (ex: chicken and rice, buttered toast, etc)"
```

Comment:
For this placeholder text let's maybe come up with a more descriptive title. Maybe something along the lines of "enter some keywords to describe your meal"

Reply:
Done.